src/databricks/labs/remorph/reconcile/connectors/data_source.py

```diff
@@ -52,7 +65,11 @@ def _get_jdbc_reader_options(options: JdbcReaderOptions):
 
     def _get_secrets(self, key_name: str):
         key = self.engine + '_' + key_name
-        return self.ws.secrets.get_secret(self.scope, key)
+        dbutils = self.ws.dbutils
```

Comment:
please convert bytes to actual string don't use dbutils directly.  using a decoder